Android/ThirdPartyAuth/app/src/main/java/com/bentley/sample/thirdpartyauth/LoginViewModel.kt

```diff
@@ -20,30 +20,30 @@ class LoginViewModel(
     
     /** Authorization client to login to Auth0 */
     private var _auth0Client: Auth0 = Auth0(
-            resourceHelper.getString(R.string.ITMSAMPLE_AUTH0_CLIENT_ID),
-            resourceHelper.getString(R.string.ITMSAMPLE_AUTH0_DOMAIN)
+            resourceHelper.getString(R.string.ITMSAMPLE_AUTH0_CLIENT_ID), // from ITMSamples.properties
```

Comment:
I wonder if these should be converted to lower case. Doing it this way kind of indicates that there's something special about the values, but also makes them look weird. Not sure which way would be better.

Reply:
It's a matter of preference I suppose. I don't mind using capitals since they are somewhat special in that they are received from a properties file